wormhole/file_transport.go

```diff
@@ -269,6 +274,12 @@ func (t *fileTransport) connectToRelay(ctx context.Context, successChan chan net
 		}
 		wsconn.SetReadLimit(websocketReadSize)
 		conn = websocket.NetConn(ctx, wsconn, websocket.MessageBinary)
+
+		err = conn.SetDeadline(time.Now().Add(time.Second * 60))
+		if err != nil {
+			failChan <- addr
+			return
+		}
 	}
 
 	_, err = conn.Write(t.relayHandshakeHeader())
```

Comment:
I think after the `Read` or `Write` you should probably add a `conn.SetDeadLine(time.Time{})` in order to remove the timeout for other operations using the same connection.

https://pkg.go.dev/net#Conn.SetDeadline